### DISCUSSION TITLE
23.10.11 데일리 스크럼

### DISCUSSION THREAD

**Discussion by Nine-JH** [petit-a-petit/MIML#27] [Daily Scrum]
.

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 할인 정책 추가.
  * 구매 및 결제에 필요한 검증 검색
* 내일은 어떤 일을 하나요?
  * 다른분들 코드리뷰
  * 검증 추가 업데이트

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - DTO 관련 공부(네이밍, 구현 방식)
- 내일은 어떤 일을 하나요?
    - 학습한 DTO 적용
    - 플레이리스트 CRUD 테스트 작성 및 구현

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - @Transactionaleventlistener 공부
  - 구현한 기능 리팩토링 진행
- 내일은 어떤 일을 하나요?
  - 피드백 받은 내용에 대한 답변 & 수정

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 코드 리뷰
    - PR (작성 중)
- 내일은 어떤 일을 하나요?
    - JWT 학습 및 적용
